### ISSUE TITLE
Hi，thank you for sharing your code. Could you tell me why did you set croppings = (target!=254).float()  in train_withdensecrfloss.py? I really don't understand what you mean by 254 here. 

### ISSUE THREAD

**Issue by yajunbaby** [meng-tang/rloss#12]

**Comment by meng-tang** [comment]
254 is for padding, see the function RandomScaleCrop in /dataloaders/custom_transforms.py and dataloaders/datasets/pascal.py.
The convention is to pad 255 and ignore the padded region. However, in scribble-annotated images, we need to distinguish ignore region and padded region for our loss function.
Sorry for the confusion.
